src/test/java/pages/HomePage.java

```diff
@@ -6,6 +6,9 @@
 import org.openqa.selenium.support.PageFactory;
 
 public class HomePage {
+    public HomePage(WebDriver driver)   {
```

Comment:
Please add `private final WebDriver driver` variable

Reply:
same as above

Reply:
We need this to initiate elements

Reply:
Ok, I agree